packages/routes/routes.0.6.0/opam

```diff
@@ -0,0 +1,36 @@
+opam-version: "2.0"
+maintainer: "Anurag Soni <anurag@sonianurag.com>"
+authors: [ "Anurag Soni <anurag@sonianurag.com>" ]
+license: "BSD-3-clause"
+homepage: "https://github.com/anuragsoni/routes"
+bug-reports: "https://github.com/anuragsoni/routes/issues"
+dev-repo: "git+https://github.com/anuragsoni/routes.git"
+doc: "https://anuragsoni.github.io/routes/"
+tags: [ "router" "http" ]
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
+]
+depends: [
+  "ocaml" {>= "4.05"}
+  "dune" { >= "1.1" }
```

Comment:
@mseri About the comment from https://github.com/ocaml/opam-repository/pull/15560

I've switched to using `(using fmt 1.0)` in the dune-project file so i think version 1.1 and above should work fine now.
Thanks again for catching this.

Reply:
`(using fmt 1.4)` has only been integrated in dune 1.4 (see https://github.com/ocaml/dune/pull/1252).

Note that you don't have to close the PR to reopen it again, adding commits to this PR is possible and probably simpler.

Reply:
https://github.com/anuragsoni/routes/blob/0.6.0/dune-project#L2 But i use `using fmt 1.0` now?

I closed the other PR as i took the opportunity to make some other changes (found a small bug at the last minute)

Reply:
Oh i see what you mean. Hmm i think i have the same dune-project for a few of my older releases too. Do those need to be fixed as well? What is the recommended way here? Should i recreate a tarball with the fixed opam file and update the PR ?

Thanks again

Reply:
> Do those need to be fixed as well?

If you have time to do it then sure, stabilizing opam is always welcome.

> What is the recommended way here? Should i recreate a tarball with the fixed opam file and update the PR ?

I'm not sure how dune-release works but you can do whatever you want for this release as it has not been merged yet (you can change it in github using git or the github interface, or retag the archive, or change the opam file locally and call dune-release, I think it might work in the same way as opam-publish), however if you want to change the other ones, changing the archive is absolutely forbidden as it would break current opam packages and would not change anything to the build itself (the opam file inside the archive does not have to do anything with opam). The way to update them is simply to clone the opam-repository, change the missing constraints and make a PR manually, without dune-release or opam-publish.

Reply:
Thank you! I'll be able to update with the proper constraints soon.